src/main/java/com/elytradev/concrete/config/ConcreteConfig.java

```diff
@@ -63,83 +85,141 @@ protected ConcreteConfig(File configFile) {
 	 * Saves config values, and writes to disk.
 	 */
 	public void loadConfig() {
-		Class clazz = this.getClass();
+		try {
+			if (fieldToDefaultValue == null) {
```

Comment:
Might make sense to move this to another method, just to make it pretty. 'cacheDefaultValue' would work.